requirements.txt

```diff
@@ -1,8 +1,11 @@
-matplotlib==3.2.1
```

Comment:
is there a reason why flake8 is not included?

Reply:
added